assets/components/src/action-card/style.scss

```diff
@@ -1,35 +1,38 @@
 /**
```

Comment:
On Mobile (iPhone XS) busy Action Cards can lead to broken layouts. 

<img width="1595" alt="Screen Shot 2019-12-11 at 11 09 51 PM" src="https://user-images.githubusercontent.com/1477002/70682023-79470f80-1c6b-11ea-8fe5-54a3dba8902a.png">

Reply:
I've changed the behaviour of the Action Card on small screens in 799d3fd (and 5f7e6d4)

![Screenshot_2019-12-16 Advertising ‹ Newspack — WordPress](https://user-images.githubusercontent.com/177929/70924553-9db33b00-2021-11ea-8911-ee8a669d2927.png)